src/equations/compressible_euler_quasi_1d.jl

```diff
@@ -109,18 +110,18 @@ as defined in [`initial_condition_convergence_test`](@ref).
     # Same settings as in `initial_condition_convergence_test`. 
     # Derivatives calculated with ForwardDiff.jl
```

Comment:
I take `ForwardDiff.derivative` as type-stable, but not sure

Reply:
👍